Userland/Services/AudioServer/ClientConnection.h

```diff
@@ -47,8 +47,10 @@ class ClientConnection final : public IPC::ClientConnection<AudioClientEndpoint,
     virtual void set_paused(bool) override;
     virtual void clear_buffer(bool) override;
     virtual Messages::AudioServer::GetPlayingBufferResponse get_playing_buffer() override;
-    virtual Messages::AudioServer::GetMutedResponse get_muted() override;
-    virtual void set_muted(bool) override;
+    virtual Messages::AudioServer::IsMainMixMutedResponse is_main_mix_muted() override;
+    virtual void set_main_mix_muted(bool) override;
```

Comment:
The declarations of these messages changed in this commit but the definitions were changed in the earlier two commits. This makes them non-atomic; if someone were to bisect some changes for possible breakage, they might hit this commit and get an unrelated compile error, which isn't very nice. Please make the definitions and implementations change in one atomic commit (perhaps move this change to the commit where you change the IPC names?).

Reply:
Okay, I'll fix this (: